packages/stylelint-config-airbnb/index.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  plugins: [
+    'stylelint-scss',
+  ],
+  rules: {
+    'at-rule-empty-line-before': 'never',
```

Comment:
ok that's just my confusion then :-) in that case, eslint shouldn't be a peer dependency.